packages/mantle/src/components/tabs/tabs.tsx

```diff
@@ -104,8 +119,14 @@ const List = forwardRef<
 			className={cx(
 				"flex border-gray-200",
 				ctx.orientation === "horizontal"
-					? "flex-row items-center gap-6 border-b"
-					: "flex-col items-end gap-[0.875rem] self-stretch border-r",
+					? "flex-row items-center gap-6"
+					: "flex-col items-end gap-3.5 self-stretch",
+				ctx.orientation === "horizontal" &&
+					ctx.appearance !== "pill" &&
+					"border-b",
+				ctx.orientation === "vertical" &&
+					ctx.appearance !== "pill" &&
+					"border-r",
```

Comment:
hmm.. we might wanna use `cva` here and in the spots below instead

this is getting hard to read and i'm worried about long-term maintenance

Reply:
it's better, but I think cva is gonna be more natural to understand

the built in variants are purpose made for this use case